django/contrib/admin/checks.py

```diff
@@ -774,8 +774,16 @@ def _check_list_editable_item(self, cls, model, field_name, label):
                                           model=model, obj=cls, id='admin.E121')
         else:
             if field_name not in cls.list_display:
-                return refer_to_missing_field(field=field_name, option=label,
-                                              model=model, obj=cls, id='admin.E122')
+                return [
+                    checks.Error(
+                        "The value of '%s' refers to %s but it is not defined in 'list_display'." % (
```

Comment:
documentation in docs/ref/checks.txt says "which is not contained in 'list_display'".